packages/playground/windows/PlaygroundNativeModules/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.210312.4" targetFramework="native" />
+  <package id="Microsoft.UI.Xaml" version="2.3.191129002" targetFramework="native" />
```

Comment:
this can be removed (it's not using WinUI, just system xaml)

Reply:
Yeah, I just copied the project from RNPicker and renamed things 😬. I failed two or three times trying to set up a fresh WinRT component project 😑